README.md

```diff
@@ -53,6 +53,36 @@ python metriq_gym/run.py poll --job_id <METRIQ_GYM_JOB_ID>
 
 where `<METRIQ_GYM_JOB_ID>` is the assigned job ID of the job that was dispatched as provided by `metriq-gym`. 
 
+### Listing jobs
+
+You can view all the jobs that have been dispatched by using the `--list-jobs` action. This will read the `jobs_file`
+and display information about each job, including its ID, backend, job type, provider, qubits, and shots.
+
+```sh
+python metriq_gym/run.py list-jobs
+```
+
+The `--list-jobs` command also supports filtering jobs based on specific criteria:
```

Comment:
We decided the action (choice: `dispatch`, `poll`, `list-jobs`) is a positional argument, that is, without `--`

Reply:
Yes, that's right. It functions that way that we decided, but I described this incorrectly here in the README. Fixed in 62d3a7b.